### ISSUE TITLE
Move version from output to a command line argument output

### ISSUE THREAD

**Issue by arturdryomov** [gojuno/mainframer#208]
Since Mainframer will be shipped as a system-wide command...

* It is possible to find out the version using the `--version` command. Bash script could have this too though, but anyway.
* `brew info mainframer` should show the same.

This way we can remove `:: mainframer v2.1.0` from the output.

@artem-zinnatullin, WDYT?

**Comment by artem-zinnatullin** [comment]
Yep, but at the same time it eases figuring out if something is wrong when user just copy-pastes the output.

Also, I think it encourages users to check new versions and keep Mainframer up-to-date :)

But I hear you, I was thinking about it too during Rust rewrite.
